src/plugins/intel_gpu/tests/unit/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -2940,7 +2940,7 @@ class fully_connected_gpu_tests: public ::testing::Test {
             auto impl = inst->get_impl();
             ASSERT_TRUE(impl != NULL);
             auto kernel_num = (is_dynamic) ? 3 : 2;
-            kernel_num = (quantize_group_size < 32) ? 2 : kernel_num;
+            kernel_num = (quantize_group_size < 32) ? ((quantize_group_size != -1) ? 2 : kernel_num) : kernel_num;
```

Comment:
it would be better to change `quantize_group_size` to uint64_t. Then you can simplify this line I guess.